patroni/ctl.py

```diff
@@ -589,9 +591,14 @@ def failover(config_file, cluster_name, master, candidate, force, dcs, scheduled
     scheduled_at = parse_scheduled(scheduled)
 
     if scheduled_at:
+        if is_paused(cluster):
+            raise PatroniCtlException("Can't schedule failover in the paused state")
         scheduled_at = scheduled_at.isoformat()
 
-    failover_value = {'leader': master, 'candidate': candidate, 'scheduled_at': scheduled_at}
+    failover_value = {'candidate': candidate, 'scheduled_at': scheduled_at}
+    if master:
```

Comment:
POST-ing {"leader": null} is absolutely the same as not posting "leader" at all. No need to change previous line.